client/stmt.go

```diff
@@ -56,18 +57,34 @@ func (s *Stmt) Close() error {
 	return nil
 }
 
+// https://dev.mysql.com/doc/dev/mysql-server/latest/page_protocol_com_stmt_execute.html
 func (s *Stmt) write(args ...interface{}) error {
+	defer clear(s.conn.queryAttributes)
 	paramsNum := s.params
 
 	if len(args) != paramsNum {
 		return fmt.Errorf("argument mismatch, need %d but got %d", s.params, len(args))
 	}
 
-	paramTypes := make([]byte, paramsNum<<1)
-	paramValues := make([][]byte, paramsNum)
+	if (s.conn.capability&CLIENT_QUERY_ATTRIBUTES > 0) && (s.conn.includeLine >= 0) {
```

Comment:
```suggestion
	if s.conn.capability&CLIENT_QUERY_ATTRIBUTES > 0 && s.conn.includeLine >= 0 {
```

the old code has clear priority. It's OK to keep it.